Dockerfile

```diff
@@ -1,34 +1,46 @@
-FROM node:12-stretch
-LABEL description="Debian image to build nativefier apps"
+FROM node:12-alpine
+LABEL description="Alpine image to build Nativefier apps"
 
-# Get wine32, not 64, to work around binary incompatibility with rcedit.
-# https://github.com/nativefier/nativefier/issues/375#issuecomment-304247033
-# Forced us to use Debian rather than Alpine, which doesn't do multiarch.
-RUN dpkg --add-architecture i386
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get --yes install wine32 imagemagick \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add bash wine imagemagick dos2unix \
+    && rm -rf /var/cache/apk/*
+
+WORKDIR /nativefier
 
 # Add sources
-COPY . /nativefier
+COPY . .
+
+# Fix line endings that may have gotten mangled in Windows
+RUN find ./icon-scripts ./src ./app -type f -print0 | xargs -0 dos2unix
 
 # Build nativefier and link globally
 WORKDIR /nativefier/app
 RUN npm install
 WORKDIR /nativefier
-RUN npm install && npm run build && npm link
+
+# Install (note that we had to manually install in `app` before, as `prepare` won't run as root)
+# Also, running tests, to ensure we don't Docker build & publish broken stuff
+RUN npm install && npm run build && npm test && npm link
+
+# Cleanup test artifacts
+RUN rm -rf /tmp/nativefier*
```

Comment:
This does not work because the files are already cached in the layers and the size won't be reduced.

Reply:
Good point! I'll combine this (and a similar one) into the same step they're generated in so they never get cached to a layer.

Reply:
I think I have a fix in #1128, could you take a look @SuperSandro2000? Thanks!